### PR TITLE
Better control timer in playtime

### PR DESCRIPTION
**Pull request by nitro2**
- [x] Fully support  `timerstop`/`timerstart`
- [x] Automatically pause and resume gameplay via `setplaytime` - TODO @neihousaigaai

### CODE REVIEW

File: game/__init__.py [hunk 1]
```diff
@@ -622,6 +624,22 @@ def is_in_play_time(self):
         else:
             return True  # a day
 
+    async def do_process_with_play_time(self):
+        if self.is_in_play_time():
+            if self.timer_stopped:
+                self.timer_stopped = False
```
Comment (nitro2) [inline game/__init__.py:630]: This will overwrite the `timerstop` command.

File: game/__init__.py [hunk 2]
```diff
@@ -319,6 +322,8 @@ async def run_game_loop(self):
         info = text_template.generate_werewolf_list(werewolf_list)
         await asyncio.gather(*[role.on_betrayer(info) for role in self.get_alive_players() if isinstance(role, roles.Betrayer)])
 
+        await self.interface.send_text_to_channel(f"Playtime: From {self.play_time_start} to {self.play_time_end} {str(tzlocal.get_localzone())}", config.GAMEPLAY_CHANNEL)
```
Comment (neihousaigaai) [inline game/__init__.py:325]: `self.play_time_start` and `self.play_time_end` are in UTC, so we may remove the `tzlocal.get_localzone()`.

File: commands/command.py [hunk 3]
```diff
@@ -143,6 +144,10 @@ async def parse_command(client, game, message):
                 )
             else:
                 timer_phase = list(map(int, parameters))
+                # Check if any timer phase is too short (<= 5 seconds):
+                if (not timer_phase) or (any (map(lambda x:x<=5,timer_phase))):
```
Comment (neihousaigaai) [inline commands/command.py:148]: remove redundant brackets (with following PEP8):
`if not timer_phase or any(map(lambda x: x<=5, timer_phase)):`

**Review by nitro2 — COMMENTED** [review]
Approve

**Pull request merged**